.swiftformat

```diff
@@ -0,0 +1,9 @@
+--indent 4
+--linebreaks lf
+--wraparguments before-first
+--wrapcollections before-first
+--commas inline
+--allman false
+--semicolons inline
+--trimwhitespace always
+--disable redundantReturn
```

Comment:
`return` statements are implicit since Swift 5.1 but since we use them explicitly, I don't want swiftformat to change the implementation for us